koku/reporting/provider/aws/models.py

```diff
@@ -154,8 +154,9 @@ class Meta:
     blended_cost = models.DecimalField(max_digits=24, decimal_places=9, null=True)
     markup_cost_blended = models.DecimalField(max_digits=33, decimal_places=15, null=True)
     savingsplan_effective_cost = models.DecimalField(max_digits=24, decimal_places=9, null=True)
-    calculated_amortized_cost = models.DecimalField(max_digits=33, decimal_places=9, null=True)
     markup_cost_savingsplan = models.DecimalField(max_digits=33, decimal_places=15, null=True)
+    calculated_amortized_cost = models.DecimalField(max_digits=33, decimal_places=9, null=True)
+    markup_cost_amortized = models.DecimalField(max_digits=33, decimal_places=9, null=True)
```

Comment:
All of these tables are being updated with these fields but I don't see any changes to the SQL to actually put values into these columns, is that coming in a separate PR?

Reply:
No, I guess I need to update some SQL too.

Reply:
Yeah, so most of these are our UI summary tables, and we actually keep a SQL file per UI summary table that has the same name as the `db_table` in the class meta. These SQL files are how we are populating those tables. For example, here is the file for the `AWSComputeSummaryByAccountP` model: https://github.com/project-koku/koku/blob/main/koku/masu/database/sql/aws/reporting_aws_compute_summary_by_account_p.sql

Reply:
Think I finally wrapped my head around all the places the SQL needs updating. I pushed up the changes but I've got more to do for OCP on AWS.